launcher/launcher/main.go

```diff
@@ -188,6 +194,33 @@ func startLauncher(launchSpec spec.LaunchSpec, serialConsole *os.File) error {
 	}
 	defer tpm.Close()
 
+	// check DA info, don't crash if failed
+	daInfo, err := launcher.GetTPMDAInfo(tpm)
+	if err != nil {
+		log.Printf("failed to get DA Info: %v\n", err)
+	} else {
+		log.Printf("TPM DA params: %+v\n", daInfo)
+
+		if !daInfo.StartupClearOrderly {
+			log.Printf("TPM didn't start orderly, DA lockout counter incremented: %d/%d\n", daInfo.LockoutCounter, daInfo.MaxAuthFail)
```

Comment:
I don't remember the details of all the startup state and clear stuff. What is this check doing

Reply:
Why are we communicating this to customers?

Also nit: this should explain the numeric values.

Reply:
If the customer use `reset` on the VM or other sudden shutdown, then the TPM is not orderly shutdown and will have the DA counter increase. This is merely telling them in the log to give a heads up